# S3EQ_of_neuronal_epigenome_and_transcriptome

## BACKGROUND:
High-throughput sequencing has been widely applied to uncover the molecular mechanisms underlying neurological and psychiatric disorders. The large body of data supports the role of epigenetic mechanisms in neurological function of both human and animals. Yet, the existing data is limited by the fact that epigenetic and transcriptomic changes have only been measured in separate cohorts, this has limited precise correlation of measured epigenetic perturbations to changes in gene expression. 

## NEW METHOD:
Single Sample Sequencing (S3EQ) is an innovative approach to analyze both epigenetic and transcriptomic regulation within a single neuronal sample. Using this method, we analyzed chromatin immunoprecipitation (ChIP)- and RNA-sequencing data from the nucleus accumbens (NAc) of the same animal. 

## RESULTS:
ChIP-S3EQ of neuronal nuclei reliably identified hPTM enrichment in the adult mouse NAc with high precision. Comparing cellular compartments, we found that the spliceosome of whole cell RNA-seq was more closely recapitulated by cytosolic RNA-S3EQ than nuclear RNA-seq. Finally, S3EQ showed increased sensitivity for correlating chromatin modifications with gene expression, especially for lowly expressed transcripts. 

## COMPARISON WITH EXISTING METHODS:
S3EQ accurately generates both RNA- and ChIP-seq from a single sample, providing a clear advantage over existing methods which require two samples. ChIP-S3EQ performance was comparable to ChIP-seq, while RNA-S3EQ generated an almost identical expression profile to nuclear and whole cell RNA-seq. Finally, we directly compared RNA-seq by cellular compartments, addressing a limitation of RNA-seq studies limited to neuronal nuclei.  

## CONCLUSION:
The S3EQ method can be applied to improve the correlative power of transcriptomic and epigenomic studies in neuronal tissue.  
